docs/src/components/home/CoreShowcase.tsx

```diff
@@ -292,7 +292,7 @@ export default function CoreShowcase() {
           >
             <Box sx={{ position: 'relative' }}>
               {startLine !== undefined && (
-                <FlashCode startLine={startLine} endLine={endLine} sx={{ mx: -1 }} />
+                <FlashCode startLine={startLine} endLine={endLine} sx={{ mx: -2 }} />
```

Comment:
@mbrookes 

> The right margin of the code preview doesn't match the rest of the copy.

This one was under the _Pricing_ list but I figure you're talking about the code preview on the home page. I'm not sure what exactly are you talking about. Could you please share some print of what is not matching? (Just commenting here so we can resolve this thread later).

Reply:
> This one wa.s under the Pricing list

Moved, thanks

> Could you please share some print of what is not matching?

It seems it's overflowing its container:

![image](https://user-images.githubusercontent.com/357702/130659201-06ae7a15-55d6-44da-b7f2-c57f68ea9c87.png)

Reply:
I think I don't know how to make this one right. There's an `overflow: 'hidden'` set on this div so it's not scrollable but it seems to not resolve the right margin there. @siriwatknp Please take a peek at this later! :)